drivers/at86rf215/Kconfig

```diff
@@ -196,27 +196,181 @@ config AT86RF215_DEFAULT_MR_FSK_MOD_IDX
+
+config AT86RF215_DEFAULT_LEGACY_OQPSK
+    bool "legacy O-QPSK"
+    depends on AT86RF215_OQPSK
+    help
+        O-QPSK compatible with IEEE 802.15.4-2003 devices
+
+config AT86RF215_DEFAULT_MR_OQPSK
+    bool "MR-O-QPSK"
+    depends on AT86RF215_MR_OQPSK
+    help
+        MR-O-QPSK according to IEEE 802.15.4g
+
+config AT86RF215_DEFAULT_MR_OFDM
+    bool "MR-OFDM"
+    depends on AT86RF215_MR_OFDM
+    help
+        MR-O-OFDM according to IEEE 802.15.4g
+
+config AT86RF215_DEFAULT_MR_FSK
+    bool "MR-FSK"
+    depends on AT86RF215_MR_FSK
+    help
+        MR-FSK according to IEEE 802.15.4g
+
+endchoice
+
+endmenu # Modulations
+
+config MODULE_AUTO_INIT_AT86RF215
+    bool "Auto-initialize the driver on startup"
+    depends on MODULE_AUTO_INIT
+    default y
+
+config MODULE_AT86RF215_BATMON
+    bool "Enable battery monitor"
+    select MODULE_SYS_BUS
+    select MODULE_SYS_BUS_POWER
+
+config AT86RF215_BATMON_THRESHOLD
+    int "Treshold voltage (in mV) of the battery monitor"
+    range 1700 3675
+    default 1800
+    depends on MODULE_AT86RF215_BATMON
+    help
+        If the supply voltage falls below the configured threshold
+        a SYS_BUS_POWER_EVENT_LOW_VOLTAGE event is generated on the
+        SYS_BUS_POWER bus.
+
+        Battery Monitoring is disabled when the device is in Deep Sleep.
+
+config AT86RF215_USE_CLOCK_OUTPUT
+    bool "Enable clock output"
+    help
+        Enable this to enable the clock output pin of the AT86RF215 chip.
+        This way it can be used as a clock source in place of a separate crystal.
+        You also have to enable this if you want to measure the clock frequency
+        for trimming. After proper trim value is applied this may be disabled
+        if not used otherwise.
+        By Default it is turned off to save energy.
+
+menuconfig AT86RF215_TRIM_VAL_EN
+    bool "Enable crystal oscillator trimming"
+    help
+        Enable crystal oscillator trimming.
+
+config AT86RF215_TRIM_VAL
+    int "Trim value for the crystal oscillator"
+    range 0 15
     default 0
+    depends on AT86RF215_TRIM_VAL_EN
     help
-        Default Settings for Forward Error Correction
+        Each increment adds 300nF capacitance between the crystal oscillator pins
+        TCXO and XTAL2.Tweak the value until the measured clock output matches
+        26 MHz the best.
+        For more information Refer Table 6-25 TRIM in Datasheet
 
-        0: No Forward Error Correction
-        1: Non-recursive and non-systematic code
-        2: Recursive and systematic code
+config AT86RF215_RPC_EN
+    bool "Enable Reduced Power Consumption"
+    help
+        Reduce Power Consumption in RX IDLE by duty-cycling the RF circuitry.
+
+config MODULE_AT86RF215_TIMESTAMP
+    bool "Enable timestamp frame information"
+
+config MODULE_AT86RF215_BLOCKING_SEND
+    bool "Block while sending"
 
-endmenu
+config MODULE_AT86RF215_SUBGHZ
+    bool
+    prompt "Sub-GHz support" if !HAVE_AT86RF215M
```

Comment:
That has it reversed - AT86RF215M is sub-GHz only

Reply:
Yeah, that is what I tried modelling here. When `HAVE_AT86RF215M` is there the prompt is not visible and defaults to `y` (i.e., the user can't de-select the sub-GHz support, which is enabled).